docs/lantern.md

```diff
@@ -0,0 +1,32 @@
+# Lantern
+
+## Overview
+
+Project Lantern is an ongoing effort to reduce the run time of Lighthouse and improve audit quality by modeling page activity and simulating browser execution.
+
+## Accuracy
+
+All of the following accuracy stats are reported excluding the 10% tail as the initial research found approximately ~10% of sites will radically vary simply by visiting the page a second time through no fault of the metrics or prediction logic. This means the accuracy is slightly overstated but should still hold for the  controlled-enivornment/repeated view use case.
+
+Stats were collected using the [trace-evaluation](https://github.com/patrickhulce/lighthouse-trace-evaluations) scripts. Table cells contain [Spearman's rho](https://en.wikipedia.org/wiki/Spearman%27s_rank_correlation_coefficient) and [MAPE](https://en.wikipedia.org/wiki/Mean_absolute_percentage_error) for the respective metric.
+
+| Comparison | FCP | FMP | TTI |
+| -- | -- | -- | -- |
+| Lantern predicting Default LH | .850 : 19.6% | .866 : 21.0% | .907 : 26.9% |
+| Lantern predicting LH on WPT | .764 : 34.4% | .795 : 32.5% | .879 : 33.1% |
+| Lantern w/adjusted settings predicting LH on WPT<sup>1</sup> | .769 : 32.9% | .808 : 31.1% | .879 : 32.6% |
```

Comment:
What settings were adjusted here?

Reply:
RTT, throughput, and CPU multipliers

I'll move the footnote that explains this to be on `adjusted settings` instead of the end of the line 👍